main/acle.rst

```diff
@@ -5822,3 +5835,195 @@ Instructions
 +---------------------------------+----------------+------------+-----------------+
 
 These intrinsics are available when ``arm_acle.h`` is included.
+
+Architectural Extension Bridges
+###############################
+
+.. _ssec-NEONSVEBridge:
+
+NEON-SVE Bridge
+===============
+
+
+The NEON_SVE Bridge adds intrinsics that allow conversions between NEON and
+SVE vectors.
+
+These intrinsics are defined in the header file ``arm_neon_sve_bridge.h``.
+
+``__ARM_NEON_SVE_BRIDGE`` is defined as 1 when ``arm_neon_sve_bridge.h`` is
+available.
+
+Including this header_file will include ``arm_neon.h`` and ``arm_sve.h``.
+
+``svset_neonq``
+---------------
+
+These intrinsics set the first 128 bits of SVE vector ``vec`` to ``subvec``.
```

Comment:
I think this is subtle enough that it would be worth expanding.

```suggestion
These intrinsics set the first 128 bits of SVE vector ``vec`` to ``subvec``.
That is, bit *i* of the result is equal to:
* bit *i* of ``subvec`` if *i* < 128
* bit *i* of ``vec`` otherwise

.. note::

   On big-endian targets, this leaves lanes in a different
   order from the “native” SVE order.  For example, if ``subvec`` is
   ``int32x4_t``, then on big-endian targets, the first memory element
   is in lane 3 of ``subvec`` and is therefore in lane 3 of the returned
   SVE vector.  Using ``svld1`` to load elements would instead put the
   first memory element in lane 0 of the returned SVE vector.
```